<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music</class>
 <widget class="QDialog" name="Music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>591</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="bt_play">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="bt_stop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="bt_quit_music">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
